app/src/main/java/com/example/next_show/data/ShowFilterer.java

```diff
@@ -26,10 +27,10 @@ public ShowFilterer () {
     private HashMap<String, List<String>> setUpFilters() {
```

Comment:
You're still not calling this anywhere though.  You should call this when you create the show filter class, right?

Reply:
Yes! I am still figuring out when/where to actually create a new show filter class in FeedFragment but this will be called when I init